LIBRARY.CSS

```diff
@@ -0,0 +1,100 @@
+
+/** THIS LIBRARY BUILD USUING CSS BLOCK CREATING TRICKS, CSS POSITIONING CSS FLEX. **/
+
+.CONTAINER{
```

Comment:
These class names should be lowercase, so that they are easier to read (i.e. `.container`)